simple/msg_pingpong.c

```diff
@@ -548,9 +548,7 @@ int main(int argc, char **argv)
 	while ((op = getopt(argc, argv, "vhg:" CS_OPTS INFO_OPTS)) != -1) {
 		switch (op) {
 		case 'v':
-			printf("%s: %s\n", argv[0], PACKAGE_VERSION);
-			printf("libfabric: %s\n", fi_tostr("1", FI_TYPE_VERSION));
-			printf("libfabric api: %d.%d\n", FI_MAJOR(FT_FIVERSION), FI_MINOR(FT_FIVERSION));
+			ft_version(argv[0]);
```

Comment:
This is a weird option for a ping-pong test to have.  I'm fine with fi_info having this test, but it really doesn't make sense for all of the tests to have it.